torchci/lib/drciUtils.ts

```diff
@@ -234,24 +234,25 @@ export async function hasSimilarFailures(
     return;
   }
 
-  // NB: It's important to sort the oldest matching results in the search window
-  // first here because that can be used to verify if the failure came from one
-  // of the previous merge commits of a reverted PR. The first record is the most
-  // relevant one and also the first time the failure is observed in the search
-  // window
+  // NB: Using the job completed_at timestamp has many FP, so it's better that
```

Comment:
Nit: consider expanding for clarity
```suggestion
  // NB: Using the job completed_at timestamp has many false positives, so it's better that
```